css/home.css

```diff
@@ -0,0 +1,83 @@
+body{
+    margin: 1rem;
+}
+
+.home-pg-container{
+   display: flex;
+   flex-direction: column;
+   justify-content:space-evenly;
+   height: 100rem;
+}
+
+.nav-title{
+    font-size: 1.5rem;
+    font-weight: bold;
+}
+
+.nav-title,h2{
+    color: var(--green);
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+.img-card-container{
+    display: flex;
+    width: 100%;
+}
+
+.img-card{
+    padding:0.5rem;
+    position:relative;
+}
+
+.img-card img{
+    opacity: 0.7;
+}
+
+.text-overlay{
+    position: absolute;
+    top:3rem;
+    left: 3rem;
+    color: var();
+    padding: 1rem;
```

Comment:
if you are not giving any color then you can delete this line

Reply:
Yes, I forgot to give color. Thanks for pointing that out.